Meadowlark-js/packages/meadowlark-core/src/plugin/backend/SearchResult.ts

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+/** TODO: this needs to be unified with GetResult */
```

Comment:
When will we do that? Do we have a natural ticket for it?

Reply:
I think this will happen as we make the backend plugin interface generic.